editor/components/AppNavBar.tsx

```diff
@@ -112,6 +98,22 @@ export default function AppNavBar({ user, logout }: NavBarProps) {
                   <FaSquareArrowUpRight className="ms-3 text-muted" />
                 </AlignedLabel>
               </Dropdown.Item>
+              <Dropdown.Divider />
+              <Dropdown.Item
+                eventKey="2"
+                onClick={() =>
+                  logout({
+                    logoutParams: {
+                      returnTo: window.location.origin + BASE_PATH,
+                    },
+                  })
+                }
+              >
+                <AlignedLabel>
+                  <FaRightFromBracket className="me-3" />
+                  Sign out
```

Comment:
I like sign out being at the bottom of the dropdown

Reply:
glad you noticed! yeah i didn't realize how much i relied on Sign Out always being the last option on the list